include/cantera/kinetics/Arrhenius.h

```diff
@@ -159,19 +158,24 @@ class ArrheniusBase : public ReactionRate
         m_negativeA_ok = value;
     }
 
+    virtual bool ready() const override {
+        return m_ready;
+    }
+
 protected:
-    bool m_negativeA_ok; //!< Flag indicating whether negative A values are permitted
-    double m_A; //!< Pre-exponential factor
-    double m_b; //!< Temperature exponent
-    double m_Ea_R; //!< Activation energy (in temperature units)
-    double m_E4_R; //!< Optional 4th energy parameter (in temperature units)
-    double m_logA; //!< Logarithm of pre-exponential factor
-    double m_order; //!< Reaction order
+    bool m_negativeA_ok = false; //!< Permissible negative A values
+    double m_A = NAN; //!< Pre-exponential factor
+    double m_b = NAN; //!< Temperature exponent
+    double m_Ea_R = 0.; //!< Activation energy (in temperature units)
+    double m_E4_R = 0.; //!< Optional 4th energy parameter (in temperature units)
+    double m_logA = NAN; //!< Logarithm of pre-exponential factor
+    double m_order = NAN; //!< Reaction order
     std::string m_A_str = "A"; //!< The string for temperature exponent
```

Comment:
There's a bit of copy-pasta here, if you don't mind fixing it.

```suggestion
    std::string m_A_str = "A"; //!< The string for pre-exponential factor
```